package.json

```diff
@@ -113,7 +113,7 @@
     "format": "prettier -c --write .",
     "format:report": "prettier -c .",
     "start": "electron .app/index.html",
-    "start:dev": "webpack serve --progress --env devServer --mode development",
+    "start:dev": "webpack serve --progress --mode development",
```

Comment:
What's up with this?

Reply:
Webpack provides a native way to detect whether we are running the dev server, so we don't also need to specify it manually.

```diff
-  const isDevServer = (env || {}).devServer === true;
+  const isDevServer = (env || {}).WEBPACK_SERVE === true;
```
See [here](https://webpack.js.org/api/cli/#cli-environment-variables) for details.